bitnami/airflow/templates/statefulset-worker.yaml

```
@@ -29,6 +29,7 @@ spec:
     {{- end }}
   template:
     metadata:
+      annotations: {{- toYaml .Values.airflow.worker.podAnnotations | nindent 8 }}
```

Please consider using the following:
```suggestion
      annotations: {{- include "airflow.tplValue" (dict "value" .Values.airflow.worker.podAnnotations "context" $) | nindent 8 }}
```